projectroles/utils.py

```diff
@@ -95,3 +109,197 @@ def get_app_names():
             else:
                 ret.append(s[0])
     return sorted(ret)
+
+
+class SidebarContent:
+    """Class for generating sidebar content for the site"""
+
+    def _is_active(self, request, link_names=None):
+        """Check if current URL is active in the sidebar."""
+        # HACK: Avoid circular import
+        from projectroles.urls import urlpatterns
+
+        if request.resolver_match.app_name != 'projectroles':
+            return False
+        url_name = request.resolver_match.url_name
+        if url_name in [u.name for u in urlpatterns]:
+            if link_names:
+                if not isinstance(link_names, list):
```

Comment:
Since we now always call this method in python instead of templates, we can get rid of this hack. Just provide the `link_names` arg as a list in each call and we don't need to do a conversion.